src/test/java/schemas/UserDto.java

```diff
@@ -0,0 +1,20 @@
+package schemas;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_DEFAULT)
+public class UserDto {
```

Comment:
This schema not enough getter for userName, password and setter for userName, password